Exercise11_files/Problem1/Exercise11Unix.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+#Align reference files with muscle
+../../../Bioinformatics/muscle3.8.31_i86darwin64 -in sigma.ref -out sigma.align
+../../../Bioinformatics/muscle3.8.31_i86darwin64 -in sporecoat.ref -out sporecoat.align
+../../../Bioinformatics/muscle3.8.31_i86darwin64 -in transporter.ref -out transporter.align
+
+#Make HMM profiles from each alignment
+../../../Bioinformatics/hmmer-3.1b2-macosx-intel/binaries/hmmbuild sigma.HMM sigma.align
+../../../Bioinformatics/hmmer-3.1b2-macosx-intel/binaries/hmmbuild sporecoat.HMM sporecoat.align
+../../../Bioinformatics/hmmer-3.1b2-macosx-intel/binaries/hmmbuild transporter.HMM transporter.align
+
+for file in *.fasta
```

Comment:
This works. Could have also done something like:

for file in *.fasta
do
	outSigma=$file.Sigma.hits
	outSpore=$file.Spore.hits
	outTrans=$file.Trans.hits
	./hmmsearch --tblout $outSigma sigma.hmm $file
	./hmmsearch --tblout $out sporecoat.hmm $file
	./hmmsearch --tblout $out transporter.hmm $file
done

cat *.hits | grep -v "#" | awk '{print $1,$3,$5}' | sed -E 's/tr\|[A-Z0-9]+\|[A-Z0-9]+_9//g' > hmmOut.txt